Documentation/coding-guidelines/project-guidelines.md

```diff
@@ -131,41 +131,26 @@ Temporary versions are at https://github.com/dotnet/corefx/blob/dev/eng/src/Tool
 - UAP F5 -> `uap-Windows_NT`
 
 ## Project configurations for VS
-For each unique configuration needed for a given library project a configuration property group should be added to the project so it can be selected and built in VS and also clearly identify the various configurations.<BR/>
+For each unique configuration needed for a given library project a configuration entry separated by a ';' should be added to the project so it can be selected and built in VS and also clearly identify the various configurations.<BR/>
 
-`<PropertyGroup Condition="'$(Configuration)|$(Platform)' == '$(OSGroup)-$(TargetGroup)-$(ConfigurationGroup)|$(Platform)'">`
-
-- Note that the majority of managed projects, currently all in corefx, $(Platform) is overridden to be AnyCPU.
+`<Configurations>netcoreapp-Unix-Debug;netcoreapp-Unix-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release;uapaot-Windows_NT-Debug;uapaot-Windows_NT-Release</Configurations>`
```

Comment:
I would keep the `$(TargetGroup)-$(OSGroup)-$(ConfigurationGroup)` placeholders in this line.  This one describes the pattern. The next 2 are concrete examples.